eos/form-factors/mesonic-processes.hh

```diff
@@ -92,6 +92,18 @@ namespace eos
         static constexpr const char * hqe_prefix = "B_s(*)->D_s(*)";
     };
 
+    struct DToPi {
```

Comment:
There is a problem here. I already added `DToPi` in the context of #552, so you have to rebase you branch on top of eos master branch, and solve the conflict.